External/Plugins/ResultsPanel/ListViewEx.cs

```diff
@@ -1,94 +1,71 @@
 ﻿using PluginCore;
-using PluginCore.Helpers;
-using PluginCore.Managers;
 using System;
 using System.Collections.Generic;
 using System.Drawing;
-using System.Drawing.Imaging;
-using System.Linq;
-using System.Text;
 using System.Windows.Forms;
 using System.Windows.Forms.VisualStyles;
 
 namespace ResultsPanel
 {
-    class ListViewEx : System.Windows.Forms.ListViewEx
+    internal class ListViewEx : System.Windows.Forms.ListViewEx
```

Comment:
Why the change of all the access modifiers to internal? Also, since you already marked the class as internal itself it won't make much difference to change the fields from public to internal.